Pod/Classes/SEGAdjustIntegration.m

```diff
@@ -18,16 +18,24 @@ - (instancetype)initWithSettings:(NSDictionary *)settings withAnalytics:(SEGAnal
         if ([self setEnvironmentProduction]) {
             environment = ADJEnvironmentProduction;
         }
+
         ADJConfig *adjustConfig = [ADJConfig configWithAppToken:appToken
                                                     environment:environment];
 
         if ([self setEventBufferingEnabled]) {
             [adjustConfig setEventBufferingEnabled:YES];
         }
+
         if ([self trackAttributionData]) {
             [adjustConfig setDelegate:self];
         }
 
+        double delayTime = [settings[@"delayTime"] doubleValue];
+        // The maximum delay start time of the adjust SDK is 10 seconds.
```

Comment:
Few things to consider here:

1. Add validator in our UI to ensure this is set to <10.
2. Use `min(delayTime, 10)` instead of bailing entirely. I think only `setDelay` should be used to see if the delay should be set or not.